unstructured/partition/common/lang.py

```diff
@@ -487,7 +487,7 @@ def apply_lang_metadata(
     if not isinstance(elements, list):
         elements = list(elements)
 
-    full_text = " ".join(e.text for e in elements if hasattr(e, "text"))
+    full_text = " ".join(e.text or "" for e in elements if hasattr(e, "text"))
```

Comment:
full_text = " ".join(e.text  for e in elements if (hasattr(e, "text") and e.text))
 maybe this is easier to read?